.github/workflows/benchmark.yml

```diff
@@ -17,9 +17,8 @@ on:
         description: 'Specifies the cluster region. Needed to retrieve cluster credentials'
         default: europe-west1-b
         required: false
-      load:
+      bechmark-load:
```

Comment:
:wrench: Let's clarify here (and in the duplicated input below) that this supports arbitrary helm arguments. Since we are no longer giving a default value, it's tricky to figure out what the format of the argument is supposed to be.

Reply:
:+1: